backend/models/SponsorshipFund.model.js

```diff
@@ -1,5 +1,9 @@
-import { Schema, model, Types } from "mongoose";
-import { SF_STATUS, ENDOWMENT_FUNDS } from "./Constants";
+const mongoose = require("mongoose");
+const Types = mongoose.Types;
+const Schema = mongoose.Schema;
+
+const ENDOWMENT_FUNDS = ["MEF", "WEEF", "ENGSOC", "DEAN_OF_ENG"];
```

Comment:
Im ok if you move these two constants to this file, but remove the duplicates in constants js then